src/main/kotlin/org/dif/utils/DIDUtils.kt

```diff
@@ -3,3 +3,8 @@ package org.dif.utils
 fun isDIDFragment(str: String) = str.contains("#")
 
 fun divideDIDFragment(str: String) = str.split("#")
+
+fun isDID(str: String): Boolean {
```

Comment:
I'm not sure this check is valid please confrim it with @andkononykhin

Reply:
@DenisRybas please check how is it [implemented](https://github.com/sicpa-dlab/didcomm-python/blob/a70250476886eb91c66e2170dbaa8df2dd1131a9/didcomm/core/utils.py#L89-L109) in python. Makes sense to add a similar TODO to cover the pattern from the spec properly later.